tutorials/javaFxPart1.md

```diff
@@ -109,6 +109,30 @@ public class Duke extends Application {
 }
 ```
 
+However, if you do already have an existing constructor which takes arguments, you can proceed 
+by creating an overloaded constructor with no arguments, passing the arguments to the existing 
+constructor e.g.,
```

Comment:
Ensure that the reader understands both: what needs to be done, and, the reason for doing so.